battle-engine.js

```diff
@@ -510,6 +506,16 @@ BattlePokemon = (function () {
 		}
 		return null;
 	};
+	BattlePokemon.prototype.isSuppressing = function (effectType) {
```

Comment:
Now that I think about it, two functions is probably best since my changes ATM break the interaction Gastro Acid Baton Pass into Delta Stream, but items need the `Start` event whitelisted for things like Choice items.

That makes me wonder, though. @Marty-D, do abilities like Cloud Nine or Mold Breaker reveal themselves if you Baton Pass Gastro Acid into them? If they do, then we'd simply need to tell the weather-setting abilities and what-not to return from their `onStart` handler early if ability is being suppressed.